recipes/ddocent/meta.yaml

```diff
@@ -0,0 +1,63 @@
+package: 
+  name: ddocent
+  version: '2.1'
+source:
+  fn: jpuritz-version-2.1.zip
+  sha256: 06f087e5a0ce094e6bea2ef0536d2605083550bdb6d74e0fc2ed99d48cba46f0
+  url: https://github.com/jpuritz/dDocent/archive/jpuritz-version-2.1.zip
+build:
+  number: 0
+requirements:
+  build:
+    - gcc #  [unix]
+    - curl
+    - cmake
+    - freebayes
+    - gawk
+    - stacks
+    - trimmomatic
+    - mawk
+    - bwa
+    - samtools 0.1.19
+    - vcftools 0.1.11 
+    - rainbow
+    - seqtk
+    - cd-hit
+    - bedtools 2.23.0
+    - vcflib
+    - gnuplot
+    - parallel
+    - bamtools
+    - java-jdk
+    - pear
+  run:
+    - libgcc # [unix]
+    - curl
+    - cmake
+    - freebayes
+    - gawk
+    - stacks
+    - trimmomatic
+    - mawk
+    - bwa
+    - samtools 0.1.19
+    - vcftools 0.1.11
+    - rainbow
+    - seqtk
+    - cd-hit
+    - bedtools 2.23.0
```

Comment:
This is an invalid syntax, isn't it? Do you need the strict dependencies or can you simply use bedtools. Same holds true for samtools and vcftools